travis.sh

```diff
@@ -53,3 +53,16 @@ fold_end
 fold_start "commandline-tests"
 yarn test
 fold_end
+
+#
+# Deploy
+#
+
+fold_start "docker-build"
+docker build -t iov-faucet:manual .
```

Comment:
pp: In other projects we use a dynamic build version based on git hash but default back to `manual` for local dev builds. see https://github.com/iov-one/weave/blob/master/.travis.yml#L22
I would therefore suggest to change this to:
```bash
BUILD_VERSION=${BUILD_VERSION:-manual}
docker build -t iov-faucet:${BUILD_VERSION} .
```
and use `iov-faucet:${BUILD_VERSION}` below

Reply:
Please let me know if you are happy with `iov-faucet` as repo name, then I would create it for you in https://hub.docker.com/u/iov1/

Reply:
Since faucet is already taken and `iov-faucet` is the name of the binary, I think the name is good

Reply:
okay, build version added